src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+/* eslint-disable react/jsx-one-expression-per-line */
+// "react/jsx-one-expression-per-line": [<enabled>, { "allow": "none"|"literal"|"single-child" }];
+// eslint-disable-next-line react/jsx-one-expression-per-line
+
+export const Person = ({ person: {
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+} }) => {
+  const gender = sex;
```

Comment:
The 'gender' variable is redundant, as it is an exact copy of the 'sex' variable. You can use the 'sex' variable directly in your code.